sdk_container/src/third_party/coreos-overlay/coreos-base/oem-ami/files/manglefs.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+
+set -euo pipefail
+
+rootfs="${1}"
+
+cat > "${rootfs}/usr/lib/systemd/system/setup-oem.service" <<-'EOF'
+[Unit]
+Description=Setup OEM
+
```

Comment:
Should this run before amazon-ssm-agent.service?
Also, would symlinks work, too?

Reply:
With `cp` the user edits also get lost. With a symlink we could check whether a custom target is set and then don't touch it and document how that works to opt-out of auto-updates for that file.